apps/styleguide/src/mixins/DefaultMenuMixin.vue

```diff
@@ -20,6 +20,10 @@ export default {
           label: "Settings",
           href: "../settings/",
         },
+        {
+          label: "Docs",
```

Comment:
maybe 'Guide' or 'Help" ?

Reply:
What do you prefer? I propose 'Help'

Reply:
I would prefer 'Guide' , heb 'Help' or 'Docs' are also fine with me